core/checkModelStruct.m

```diff
@@ -246,6 +246,18 @@ function checkModelStruct(model,throwErrors,trimWarnings)
 EM='The following reactions have bounds contradicting their reversibility:';
 dispEM(EM,throwErrors,model.rxns(model.lb<0 & model.rev==0),trimWarnings);
 
+%Multiple or no objective functions not allowed in SBML L3V1 FBCv2
+if find(model.c)>1
+    EM='The model has multiple objective functions, which might be intended, but will not allow export to SBML';
```

Comment:
```suggestion
    EM='The model has multiple objective functions, which might be intended, but not compliance with SBML FBCv2';
```

Reply:
I on purpose mentioned model export, because this issue throws a warning (it should not be an error, as it is not necessarily an invalid model, there are situations where multiple objective functions make sense), but it does cause `exportModel` to fail.

What about "... which will fail model export due to non-compliance with SBML FBCv2"?

Reply:
sounds good to me

Reply:
Resolved in 739ea86c49a49d036829c1a20b4d23143b4a157f